Client/src/Components/Profile/ProfileGraphRow.tsx

```diff
@@ -1,16 +1,30 @@
-import { Box, Grid, Typography } from "@material-ui/core"
-
+import { Box, Grid, Typography, IconButton } from "@material-ui/core"
 import { IGraphStateModel } from '../../Models/Graph/IGraphStateModel'
 import { Link } from 'react-router-dom'
-import React from 'react'
+import React, { useState } from 'react'
 import { classStyles } from '../../appTheme'
+import CancelIcon from '@material-ui/icons/Cancel'
+import { ConfirmationModal } from '../Authentication/ConfirmationModal'
+import { callDeleteGraphState } from "../../Remote/Endpoints/GraphStateEndpoint"
+import SnackbarUtils from '../../Components/Utils/SnackbarUtils'
+
 
 interface IGraphModel {
     graphset: IGraphStateModel,
+    handleRemoveGraphSet: (graphState: IGraphStateModel) => void,
+    userID: number
 }
 
 export const ProfileGraphRow = (props: IGraphModel) => {
-    const { graphset } = { ...props }
+    const { graphset, userID } = { ...props }
+    const [confirmModalOpen, setConfirmModalOpen] = useState(false)
+
+    const onHandleRemoveGraphSet = async (removeGraphSetID: IGraphStateModel, userId: number) => {
+        setConfirmModalOpen(true)
+        await callDeleteGraphState(graphset, userId)
+        SnackbarUtils.success('Saved graph successfully deleted!')
```

Comment:
move this snackbar norification inside the callDeleteGraphState, also add a check if the result was indeed successful